dependencies.yaml

```
@@ -474,16 +474,16 @@ dependencies:
     common:
       - output_types: [conda]
         packages:
-          - pyarrow>=14.0.0,<19.0.0a0
+          - pyarrow>=14.0.0,<=19.0.0
```

Let's use the next major version as an upper bound, to permit `19.x.y` releases.
```suggestion
          - pyarrow>=14.0.0,<20.0.0a0
```

---

good point!